### DISCUSSION TITLE
[预览] v2.9.4

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4971] [分享 / Share]
包含 [v2.9.4](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.4) 的所有更新内容.

✨新增
- 新增组件 `隐藏头像框`.
> 隐藏页面中用户的头像框 (包括角标), 目前支持动态和视频页面.

- 新增组件 `隐藏直播马赛克`. (#4634)
> 移除直播画面中的马赛克区域.

- 优化了搜索框的搜索建议精准度. (#4833)
- 新增组件 `删除动态`. (PR #4915 by [gouzil](https://github.com/gouzil))
> 删除动态, 可选转发抽奖, 和全部删除.

- `保存视频元数据` 增加 FFMETADATA 字段选项. (PR #4943 by [WakelessSloth56](https://github.com/WakelessSloth56))
- 新增组件 `评论内容替换`. (#4072)
> 替换评论中的内容.
> 可以添加多个替换配置, 每项配置可将一个关键词替换为其他文本. 若替换的目标是一个链接, 则视作替换为表情.
>

- `直播信息扩充` 更换新的 API 接口. (PR #4964 by [Oxygenくん](https://github.com/oxygenkun))

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.4-preview'>[预览] v2.9.4</a>.</em>

**Comment by MonicaQvQ** [comment]
关于马赛克有两个组件了（`删除直播马赛克遮罩`和`隐藏直播马赛克`），是不是应该去掉一个，其中第一个好像不起作用了。
另外想问一下，`视频内显示时间`的Commit为什么一直都不随版本更新呀？

**Reply by WakelessSloth56** [reply]
`视频内显示时间` 是第三方组件（ https://github.com/FoundTheWOUT/Bilibili-Evolved/blob/preview/registry/dist/components/video/player/localtime.js ）。

https://github.com/the1812/Bilibili-Evolved/blob/dbd813b10041aeec0a4e1867be4a4f75c835bd5b/registry/lib/docs/third-party.ts#L6-L12

**Reply by MonicaQvQ** [reply]
原来是这样，感谢回复~

**Reply by the1812** [reply]
（忘了之前有人实现过了）
新加的这个隐藏马赛克是我用 CSS 实现的，应该稳定一些

**Reply by MonicaQvQ** [reply]
> （忘了之前有人实现过了） 新加的这个隐藏马赛克是我用 CSS 实现的，应该稳定一些

OK~谢谢！

**Reply by Liki4** [reply]
我猫了一眼看到好像还生效（因为只看 OW 直播，之前就拿了些 OW 直播间来测试 orz）
随机挑选的直播间地址：https://live.bilibili.com/3878101
![image](https://github.com/user-attachments/assets/57c8fac8-95de-4322-bd54-7e6c3f820789)
![image](https://github.com/user-attachments/assets/e7474e6b-b341-40cc-ac75-3a0c2d13acf4)

![image](https://github.com/user-attachments/assets/6350e785-24d2-431b-bc0e-90384ab4d46a)
不知道怎么回事，雪狗的直播间就没有遮罩，我还以为啊 b 把马赛克删了

有没有其他测试用的直播间我有空研究一下
![image](https://github.com/user-attachments/assets/4642d6ac-53db-4a42-9e8e-f0723aef18c4)

**Reply by MonicaQvQ** [reply]
对不起~ <(＿　＿)> 我当时应该是忘记刷新了。我用你提供的直播间测试了一下是生效的！我平时是看LOL的，LOL之前有遮罩，现在好像都没有了。
